proto/proto/kvrpcpb.proto

```diff
@@ -155,88 +168,94 @@ message CheckTxnStatusRequest {
     uint64 current_ts = 4; // current_ts is used to check TTL timeout, it may be inaccurate.
 }
 
-enum Action {
-    NoAction = 0;
-    TTLExpireRollback = 1;
-    LockNotExistRollback = 2;
-}
-
 message CheckTxnStatusResponse {
     errorpb.Error region_error = 1;
     // Three kinds of txn status:
     // locked: lock_ttl > 0
     // committed: commit_version > 0
-    // rolled back: lock_ttl = 0 && commit_version = 0
+    // rolled back: lock_ttl == 0 && commit_version == 0
     uint64 lock_ttl = 2;
     uint64 commit_version = 3;
-    // The action performed by the CheckTxnStatus
-    // If the action is rollback, set the rollback reason.
+    // The action performed by TinyKV in response to the CheckTxnStatus request.
     Action action = 4;
 }
 
+// Resolve lock will find all locks belonging to the transaction with the given start timestamp.
+// If commit_version is 0, TinyKV will rollback all locks. If commit_version is greater than
+// 0 it will commit those locks with the given commit timestamp.
+// The client will make a resolve lock request for all secondary keys once it has successfully
+// committed or rolled back the primary key.
 message ResolveLockRequest {
     Context context = 1;
     uint64 start_version  = 2;
-    // If the txn is rolled back, do not set it.
     uint64 commit_version = 3;
 }
 
+// Empty if the lock is resolved successfully.
 message ResolveLockResponse {
     errorpb.Error region_error = 1;
     KeyError error = 2;
 }
 
-message RawGetRequest {
-    Context context = 1;
+// Utility data types used by the above requests and responses.
+
+// Either a key/value pair or an error for a particular key.
+message KvPair {
+    KeyError error = 1;
     bytes key = 2;
-    string cf = 3;
+    bytes value = 3;
 }
 
-message RawGetResponse {
-    errorpb.Error region_error = 1;
-    string error = 2;
-    bytes value = 3;
-    bool not_found = 4;
+enum Op {
+    Put = 0;
+    Del = 1;
+    Rollback = 2;
```

Comment:
can you help me add a type `Lock = 3; // used by TinySQL to add index, though it's won't implemented by TinyKV`

Reply:
Sure!